test/test_cursor.py

```diff
@@ -1084,8 +1085,12 @@ def test_concurrent_close(self):
 
         def iterate_cursor():
             while cursor.alive:
-                for _doc in cursor:
-                    pass
+                try:
+                    for _doc in cursor:
+                        pass
+                except OperationFailure as e:
+                    if e.code != 237:
```

Comment:
Could you add a comment saying this is the CursorKilled error code?

Reply:
Done.